frontend/admin/package.json

```diff
@@ -93,6 +100,7 @@
     "dotenv": "^14.3.2",
     "graphql": "^16.3.0",
     "history": "^5.2.0",
+    "jwt-decode": "^3.1.2",
```

Comment:
Just noting that 0.8kb of the 0.85kb bundlesize change is from this package, but a cursory search confirmed this to be the smallest package for for doing this work with jwts :)